lib/phoenix_live_view/html_formatter.ex

```diff
@@ -179,9 +187,65 @@ defmodule Phoenix.LiveView.HTMLFormatter do
   or after the element. Otherwise it would compromise what is rendered adding
   an extra whitespace.
 
-  This is the list of inline elements:
-
-  https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements#list_of_inline_elements
+  The formatter will consider these tags as inline elements:
+
+  - `<a>`
```

Comment:
The old MDN docs were just like this, a rather long list. If maintainers prefer, we could also condense this long list into some "inline" format, like a comma or space-separated list. We could also remove the angle brackets. E.g. the code-level definition has much less noise:

```elixir
@inline_tags ~w(a abbr acronym ...
```

https://web.archive.org/web/20220405120608/https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements#list_of_inline_elements